test/integration/chunkers/test_chunkers.py

```diff
@@ -29,14 +29,3 @@ async def test_chunker_api(chunker_file: Path, strategy: str):
     chunker = Chunker(config=chunker_config)
     results = await chunker.run_async(elements_filepath=chunker_file)
     assert results
-
-
-@pytest.mark.parametrize("chunker_file", chunker_files, ids=[path.name for path in chunker_files])
-@pytest.mark.parametrize("strategy", ["basic", "by_title"])
-def test_chunker_basic(chunker_file: Path, strategy: str):
```

Comment:
any way this can be recreated in the original Unstructured repo or somewhere, if it's being pesky here due to nltk? It's always hard to give up tests